tests/check/check_h26xsigned_sign.c

```diff
@@ -673,6 +673,49 @@ START_TEST(two_completed_seis_pending)
 }
 END_TEST
 
+/* Test description
+ * Verify that after golden SEI is generated
+ * The operation is as follows:
+ * 1. Setup a signed_video_t session
+ * 2. Generate a golden SEI
+ * 3. Verify the golden SEI
+ */
```

Comment:
I think you can compress the test description to something like
"Generates a golden SEI and fetches it from the library. Then verifies that the correspond flag is set."

It could be a good idea to check that the PUBLIC_KEY tag is present as well, if so, add that info to the description.